packages/fuels-contract/src/contract.rs

```diff
@@ -571,14 +596,14 @@ where
 
     /// Call a contract's method on the node, in a state-modifying manner.
     pub async fn call(self) -> Result<CallResponse<D>, Error> {
-        Ok(Self::call_or_simulate(self, false).await?)
+        Self::call_or_simulate(self, false).await
```

Comment:
why did you remove the `?` ? Maybe I don't understand how the `?` works haha

Reply:
`call_or_simulate()` already returns the same type, so the old code is redundant